css-fonts/Fonts.src.html

```diff
@@ -4469,17 +4469,15 @@ <h3 id="om-fontface">The <code>CSSFontFaceRule</code> interface</h3>
 
 <pre class="idl">
 interface CSSFontFaceRule : CSSRule {
-  attribute DOMString family;
-  attribute DOMString src;
-  attribute DOMString style;
-  attribute DOMString weight;
-  attribute DOMString stretch;
-  attribute DOMString unicodeRange;
-  attribute DOMString variant;
-  attribute DOMString featureSettings;
+  // Intentionally left blank in this specification.
 }</pre>
 
-<p>The DOM Level 2 Style specification [[DOM-LEVEL-2-STYLE]] defined a different variant of this rule.  This definition supercedes that one.</p>
+<p class="note">The CSS Working Group would like <code>CSSFontFaceRule</code> to have
```

Comment:
Do we really want to describe the aspirations of the WG? Shouldn't we just state the reason there are no attributes in the object?